int32 xLeft
int32 xRight
int32 yFront
int32 yBack
int32 zTop
int32 zBottom
